flink-table/flink-table-common/src/main/java/org/apache/flink/table/utils/print/TableauStyle.java

```diff
@@ -154,9 +164,26 @@ public void print(Iterator<RowData> it, PrintWriter printWriter) {
 
         // print border line
         printBorderLine(printWriter);
-        final String rowTerm = numRows > 1 ? "rows" : "row";
-        printWriter.println(numRows + " " + rowTerm + " in set");
+        return numRows;
+    }
+
+    public boolean printEmptySet(Iterator<RowData> it, PrintWriter printWriter) {
```

Comment:
Do we need to keep this? I think we can revert the refactor. It's better to open a new PR to do this if you need